examples/research_projects/unilm/layoutlmv2/README.md

```diff
@@ -0,0 +1,116 @@
+<!---
+Copyright 2020 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# LayoutLMv2
+
+## Using Huggingface Trainer
+
+Fine-tuning the LayoutLMv2 library model. The main script `run_layoutlmv2.py` leverages the 🤗 Datasets library and the Trainer API. You can easily
+customize it to your needs if you need extra processing on your datasets.
+
+It will either run on a datasets hosted on our [hub](https://huggingface.co/datasets) or with your own text files for
+training and validation, you might just need to add some tweaks in the data preprocessing.
+
+The following example fine-tunes LayoutLMv2 model on FUNSD dataset:
+
+```bash
+python run_layoutlmv2.py \
+        --model_name_or_path microsoft/layoutlmv2-base-uncased \
+        --processor_name microsoft/layoutlmv2-base-uncased \
+        --output_dir /tmp/test-layoutlmv2 \
+        --dataset_name nielsr/funsd \
+        --do_train \
+        --do_predict \
+        --max_steps 1000 \
+        --warmup_ratio 0.1 \
+        --fp16 \
+        --model_revision no_ocr
+```
+
+or just can just run the bash script `run.sh`.
+
+To run on your own training and validation files, use the following command:
```

Comment:
I would add some more information here regarding how users should prepare their own data files for the model.